packages/ui/components/pages/Fuse/FusePoolPage/BorrowList/index.tsx

```diff
@@ -113,37 +113,37 @@ export const BorrowList = ({ assets, borrowBalanceFiat, comptrollerAddress }: Bo
         <Tbody>
           {assets.length > 0 ? (
             <>
-              {borrowedAssets.map((asset, index) => {
+              {borrowedAssets.map((asset) => {
                 return (
                   <AssetBorrowRow
                     comptrollerAddress={comptrollerAddress}
                     key={asset.underlyingToken}
-                    assets={borrowedAssets}
```

Comment:
In these lines only a slice of the actual `assets` were passed. Which causes some calculations later down the line to be off, as they did not have access to all of the assets of the pool.